01-ruby-fundamentals/exercises/while-worksheet.md

```diff
@@ -0,0 +1,55 @@
+# While Loop Evaluation Worksheet
+For this exercise, you will be evaluating some Ruby code and coming up with scenarios for how the code will execute.
+
+1. For the code below, describe what the loop will do with the following input to the `word` variable. Be sure to include how many times the loop will execute.
+  1. "ham"
+  2. "hamster"
+  3. "bananas"
```

Comment:
What if you gave a sequence of words for each run, and called out explicitly that not all words will necessarily be used?